src/_variables.scss

```diff
@@ -25,6 +25,8 @@
   --rqg-color-enc-legend: #e4cc9d;
 
 
+  --rqg-color-invalid-joural: #e4b5ad; //Regular journal in chat card is: #7a7971
```

Comment:
Misspelled ...-journal

Reply:
Yeah, I noticed that too.  Done.

Reply:
I might have put this reply on the wrong comment earlier!